examples/data-objects/codemirror/src/index.ts

```diff
@@ -5,14 +5,56 @@
 
 import { IContainerContext } from "@fluidframework/container-definitions";
 import { ContainerRuntime } from "@fluidframework/container-runtime";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
 import { IFluidDataStoreFactory } from "@fluidframework/runtime-definitions";
-import { buildRuntimeRequestHandler } from "@fluidframework/request-handler";
-import { defaultRouteRequestHandler } from "@fluidframework/aqueduct";
-import { RuntimeFactoryHelper } from "@fluidframework/runtime-utils";
-import { fluidExport as smde } from "./codemirror";
+import { buildRuntimeRequestHandler, RuntimeRequestHandler } from "@fluidframework/request-handler";
+import { mountableViewRequestHandler } from "@fluidframework/aqueduct";
+import {
+    requestFluidObject,
+    RequestParser,
+    RuntimeFactoryHelper,
+} from "@fluidframework/runtime-utils";
+import { MountableView } from "@fluidframework/view-adapters";
+import {
+    CodeMirrorComponent,
+    SmdeFactory,
+} from "./codeMirror";
+import {
+    CodeMirrorView,
+} from "./codeMirrorView";
+
+export {
+    CodeMirrorComponent,
+    SmdeFactory,
+} from "./codeMirror";
+export {
+    CodeMirrorView,
+} from "./codeMirrorView";
 
 const defaultComponentId = "default";
 
+const smde = new SmdeFactory();
+
+type ViewCallback<T> = (fluidModel: T) => any;
+
+const makeViewRequestHandler = <T>(viewCallback: ViewCallback<T>): RuntimeRequestHandler =>
+    async (request: RequestParser, runtime: IContainerRuntime) => {
+        if (request.pathParts.length === 0) {
+            const objectRequest = RequestParser.create({
+                url: ``,
+                headers: request.headers,
+            });
+            const fluidObject = await requestFluidObject<T>(
+                await runtime.getRootDataStore(defaultComponentId),
+                objectRequest);
+            const viewResponse = viewCallback(fluidObject);
+            return { status: 200, mimeType: "fluid/view", value: viewResponse };
+        }
+    };
+
+const viewCallback =
+    (codeMirror: CodeMirrorComponent) => new CodeMirrorView(codeMirror.text, codeMirror.presenceManager);
+
 class CodeMirrorFactory extends RuntimeFactoryHelper {
```

Comment:
@anthony-murphy CodeMirror and ProseMirror are reasonably good examples of non-aqueduct runtime factories and data object factories (related to your comment on the shared-text PR)